.gitlab-ci.yml

```diff
@@ -415,7 +415,7 @@ fuzz-tests:
   <<:                              *test-refs
   variables:
     # The QUICKCHECK_TESTS default is 100
-    QUICKCHECK_TESTS:              20000
+    QUICKCHECK_TESTS:              5000
```

Comment:
Why is this being reduced? What kind of time savings are we looking at?

Reply:
See the extended commit message. Current duration is ~15 minutes for this stage, I'd like to see how this reduces with this lower number.